projects/hello_world/src/main.c

```diff
@@ -0,0 +1,7 @@
+#include "log.h"
+
+int main(void) {
+  LOG_DEBUG("Hello World!\n");
+
+  return 0;
+}
```

Comment:
Please don't include your hello world files in homework submissions; you can remove them by just deleting them from your local repository.